docs/applications/node.md

```diff
@@ -11,3 +11,12 @@ that need to connect to the P2P network.
 This node is currently only released as a library, and not as a binary.
 This is because it (nor the documentation) is not deemed production
 ready. Use at your own risk, and without too much money depending on it.
+
+## Node Callback Handling
```

Comment:
i believe you need to rebase onto master and then put this in the correct file. We've moved things since this PR was opened